auth/permission.cc

```diff
@@ -21,7 +21,8 @@ const auth::permission_set auth::permissions::ALL = auth::permission_set::of<
         auth::permission::SELECT,
         auth::permission::MODIFY,
         auth::permission::AUTHORIZE,
-        auth::permission::DESCRIBE>();
+        auth::permission::DESCRIBE,
+        auth::permission::EXECUTE>();
```

Comment:
A comment on the commit message because I can't write it there:

You make it seem that only EXECUTE can apply to a specific FUNCTION f, and all other operations can only take ALL FUNCTIONS or ALL FUNCTIONS IN KEYSPACE. But actually, according to the Cassandra documentation, FUNCTION f can applies to all operations except CREATE - it can apply to EXECUTE, DROP, ALTER, AUTHORIZE.

Reply:
I see in the last patch that your test acknowledges that FUNCTION f can be applied to all of EXECUTE, DROP, ALTER and AUTHORIZE - not just EXECUTE. So there's nothing wrong in your code - just the commit message.

Reply:
I fixed the commit message in #12869